src/amo/components/Search/index.js

```diff
@@ -115,100 +116,104 @@ export class SearchBase extends React.Component<InternalProps> {
   }
 
   renderHelmetTitle() {
-    const { i18n, filters } = this.props;
+    const { i18n, filters, pageTitle } = this.props;
 
-    let title = i18n.gettext('Search results');
+    let title = pageTitle;
 
-    if (filters.promoted) {
```

Comment:
FYI, the way GitHub has interpreted this diff, makes it looks like there are a lot of changes, but in fact the code below is completely unchanged. The only change is to introduce the use of the `pageTitle` prop as an override.